### ISSUE TITLE
Poppet shelf crashing server

### ISSUE THREAD

**Issue by Thebullfighter9** [MoriyaShiine/bewitchment#389]
### Mod version

1.20.1

### A description of the issue

When the poppet shelf is place it seems to crash the server it has never done this before, I am not the server owner so i dont have the logs

07.01 23:35:19 [Server] Server thread/ERROR Encountered an unexpected exception
07.01 23:35:19 [Server] INFO java.lang.NoSuchFieldError: clientInventory
07.01 23:35:19 [Server] INFO at moriyashiine.bewitchment.common.block.entity.PoppetShelfBlockEntity.<init>(PoppetShelfBlockEntity.java:32) ~[bewitchment-1.20-6.jar:?]

**Comment by Cynthaisy** [comment]
I'm having the same issue in 1.19.2, here's the logs.
https://gist.github.com/Cynthaisy/d5fa644fc6366737d7cb8faab54bedf5

**Comment by AlkyoneAoide** [comment]
Had this same issue, disabling WTHIT server-side solved it.

**Comment by Cynthaisy** [comment]
> Had this same issue, disabling WTHIT server-side solved it.

Don't have WTHIT but I did have Jade. Disabling Jade caused it to not crash on join, but it continues to crash when it's placed, destroyed, or interacted with, requiring a server restart for people to be able to join again. Tried readding bewitchment too, but I keep getting the same issues. Is yours crashing on interaction too? Maybe I missed some Jade config files or something.
